docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   postgres:
-    container_name: postgres-transcendence
+    container_name: postgres-transcendence-${USER}
```

Comment:
Added our login in container name so there's one less overlap when we develop on the SSH server. Ports still overlap for now, need to change them manually